cmd/virt-v2v/entrypoint.go

```diff
@@ -2,279 +2,76 @@ package main
 
 import (
 	_ "embed"
-	"encoding/json"
-	"errors"
 	"fmt"
-	"io"
 	"os"
-	"os/exec"
-	"strconv"
-	"strings"
 
-	"github.com/konveyor/forklift-controller/pkg/virt-v2v/customize"
-	"github.com/konveyor/forklift-controller/pkg/virt-v2v/global"
+	"github.com/konveyor/forklift-controller/pkg/virt-v2v/config"
+	"github.com/konveyor/forklift-controller/pkg/virt-v2v/conversion"
 	"github.com/konveyor/forklift-controller/pkg/virt-v2v/server"
 	utils "github.com/konveyor/forklift-controller/pkg/virt-v2v/utils"
 )
 
 func main() {
-	var err error
-	if err = virtV2VPrepEnvironment(); err != nil {
-		fmt.Println("Failed to prepare the environment", err)
+	env := &config.AppConfig{}
+	err := env.Load()
+	if err != nil {
+		fmt.Println("Failed to load variables", err)
 		os.Exit(1)
 	}
-
-	// virt-v2v or virt-v2v-in-place
-	if _, found := os.LookupEnv("V2V_inPlace"); found {
-		err = runVirtV2vInPlace()
-	} else {
-		err = runVirtV2v()
+	if err = virtV2VPrepEnvironment(env); err != nil {
+		fmt.Println("Failed to prepare the environment", err)
+		os.Exit(1)
 	}
+	convert, err := conversion.NewConversion(env)
 	if err != nil {
-		fmt.Println("Failed to execute virt-v2v command:", err)
+		fmt.Println("Failed prepare conversion", err)
 		os.Exit(1)
 	}
 
-	// virt-v2v-inspector
-	var disks []string
-	disks, err = utils.GetLinkedDisks()
+	// virt-v2v or virt-v2v-in-place
+	err = convert.ConvertGuest()
```

Comment:
hmm... IDK, doe it make sense to be explicit here, doing the 
`if c.IsInPlace { ...
`
here ? it will be more clear what happen here ... ?